specification/portal/resource-manager/Microsoft.Portal/preview/2015-08-01-preview/portal.json

```diff
@@ -413,19 +413,19 @@
           "description": "The dashboard's part position.",
           "properties": {
             "x": {
-              "type": "number",
+              "type": "integer",
```

Comment:
>integer [](start = 23, length = 7)

`number` is more permissive than `integer`. Do you have any dashboard parts in existence that used non-integer numbers for any of the properties affected by this change? If yes, then this is a breaking change across 3 different API versions.